### PR TITLE
added styling and to-the-top functonality

### PR DESCRIPTION
**Pull request by cfbuckley14**

### CODE REVIEW

**Comment by maxwell-01** [comment]
consider switching your nav so that the left menu items sit above the logo and the right head items sit below (use flex column and stick each set inside a different div) 
image

**Comment by maxwell-01** [comment]
portflio card image still flowing outside the container, consider setting width to 100% and constraining the div (it would look worse with an image that doesnt have a transparent bg)

image

**Comment by maxwell-01** [comment]
You still have blue lines under your footer links, use this to solve:
```
a {
    text-decoration: none;
}
```

image

**Comment by maxwell-01** [comment]
you should `git rm filler.png` to remove it from your repo now its not being used

File: index.html [hunk 1]
```diff
@@ -3,87 +3,82 @@
...
 		<link rel="stylesheet" type="text/css" href="normalize.css">
+		<link rel="stylesheet" type ="text/css" href="style.css"> 
 		<link rel="preconnect" href="https://fonts.googleapis.com">
 		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
 		<link href="https://fonts.googleapis.com/css2?family=Varela+Round&display=swap" rel="stylesheet">
 	</head>	
 	
 	<body>		
-		<nav class="nav-bar">
+		<nav class="nav-bar" id="top">
```
Comment (maxwell-01) [inline index.html]: you could be more semantic with the naming of this, this is describing where the element _is_ not what it `does`, consider something like 'nav-bar'

File: index.html [hunk 2]
```diff
@@ -3,87 +3,82 @@
...
 			<a class="navigation-links" href="#home">home</a> 
 			<a class="navigation-links" href="#about-me">about me</a>
-			<button class="cv-button">download my CV!</button>
+			<img src="c.png" alt="CB Logo">
```
Comment (maxwell-01) [inline index.html]: alt text not really describing the image, how about 'letters cb entwined with beautiful flowers'

File: index.html [hunk 3]
```diff
@@ -3,87 +3,82 @@
...
 			<a class="navigation-links" href="#portfolio">portfolio</a>
 			<a class="navigation-links" href="#contact-me">contact me</a>	
 		</nav>	
 	
 		<header id="home">
```
Comment (maxwell-01) [inline index.html]: id name could be more semantic, remember the names need to describe what the element functionally does on the page, so id='header-section' would be more appropes

File: index.html [hunk 4]
```diff
@@ -3,87 +3,82 @@
...
-			<div>
-				<span><img class="sparkle-inline" src="sparkles.gif" alt="sparkle">hi there</span> 
+			<div class="sparkle-parent">
+				<p class="sparkle-item"><img src="sparkles.gif" alt="sparkle"/>hi there</p> 
 			</div>
-			<div>
 				<h1>welcome to my portfolio!</h1>
```
Comment (maxwell-01) [inline index.html]: this needs to be one layer less indented

File: index.html [hunk 5]
```diff
@@ -3,87 +3,82 @@
...
-			</div>
+			<div class="back-to-top">
```
Comment (maxwell-01) [inline index.html]: class="back-to-top-button" might be more semantic ?

File: index.html [hunk 6]
```diff
@@ -3,87 +3,82 @@
...
+				<a href="#top">to the top!</a>
+			</div>	
 		</header>
-		<section class="about-section" id="about-me">
+		<section id="about-me" class="about-section">
```
Comment (maxwell-01) [inline index.html:38]: this looks good! Use the same convention for your other sections :)

File: index.html [hunk 7]
```diff
@@ -3,87 +3,82 @@
...
 			<div>
-				<h2>
-					<a class="about-me-title">About me</a>
-				</h2>
-				<p class="about-me-text">Hello, I'm Charlotte - a software developer supreme in the making! I've previously worked within the Events Industry so delivering projects for clients is in my nature.<br> If you'd like to find out more, be sure to connect with me through my socials - I love a chin-wag!</p>
+				<h2 class=about-me-title>Hello, I'm Charlotte</h2>
```
Comment (maxwell-01) [inline index.html]: this doesn't need a class on it, select using `.about-section h2`

File: index.html [hunk 8]
```diff
@@ -3,87 +3,82 @@
...
+				<p class="about-me-text"> I'm a software developer supreme in the making! I've previously worked within the Events Industry so delivering projects for clients is in my nature.<br> I'm a dog person (important information to know), I grew up in a football-mad household which inevitably meant I played football, and I really enjoy baking in my spare time.<br> If you'd like to find out more, be sure to connect with me through my <a href="#contact-me">socials</a> - I love a chin-wag!</p>
```
Comment (maxwell-01) [inline index.html]: you could select this with `.about-section p` and do away with the extra class

File: index.html [hunk 9]
```diff
@@ -3,87 +3,82 @@
...
 			
 		</section>	
 	
 		<section class="portfolio-section" id="portfolio">
-			<div>
-				<h2 class="portfolio-title">
-					<a>Portfolio</a>
-				</h2>
-			</div>
+			<h2 class="portfolio-title">Portfolio</h2>
```
Comment (maxwell-01) [inline index.html]: same here, use `.portfolio-section h2`

File: index.html [hunk 10]
```diff
@@ -3,87 +3,82 @@
...
 			
 			<div class="portfolio-parent">
-				<div class="portfolio-child">
+				<div class="portfolio-item">
 					<h3>Project One</h2>
```
Comment (maxwell-01) [inline index.html]: you have h3 opening tags but h2 closing tags

File: index.html [hunk 11]
```diff
@@ -3,87 +3,82 @@
...
 					<p>Coming soon...</p>
-					<img class="under-construction" src="under-construction-lex.gif" alt="under-construction-gif">
+					<img class="under-construction" src="under-construction-lex.gif" alt="under-construction-gif"/>
```
Comment (maxwell-01) [inline index.html]: the class name should not describe the image, consider `project-card-image`

File: index.html [hunk 12]
```diff
@@ -3,87 +3,82 @@
...
 				</div>
 			</div>
 		</section>
 	
 		<footer>
-			<h3>
-				<a id="contact-me">Want to get in touch? Here's the best ways to...</a>
+			<h3 id="contact-me">Want to get in touch? Here's the best ways to...
```
Comment (maxwell-01) [inline index.html]: again here no need for a class, use `footer h3`

**Review by maxwell-01 — CHANGES_REQUESTED** [review]
please see comments about html and give me a shout when updated and ill take a look at the CSS

File: style.css [hunk 13]
```diff
@@ -207,9 +248,35 @@ footer h3 {
...
 		height: auto;
 		font-size: 1.5rem;
 	}
 }
+
+@media screen and (max-width: 1100px) {
+	.about-section h2 {
+		font-size: 3rem;
+	}
+	.about-section h3 {
```
Comment (maxwell-01) [inline style.css]: I don't think you have an h3 in your about section

File: index.html [hunk 14]
```diff
@@ -12,51 +12,62 @@
 	</head>	
 	
 	<body>		
-		<nav class="nav-bar">
-			<a class="navigation-links" href="#home">home</a> 
-			<a class="navigation-links" href="#about-me">about me</a>
-			<button class="cv-button">download my CV!</button>
-			<a class="navigation-links" href="#portfolio">portfolio</a>
-			<a class="navigation-links" href="#contact-me">contact me</a>	
+		<nav class="nav-bar" id="nav-bar">
```
Comment (maxwell-01) [inline index.html]: you dont need a class here, just select using `nav`

File: style.css [hunk 15]
```diff
@@ -124,10 +156,10 @@ header div h1 {
...
 	grid-template-rows: 25%, 25%, 25%;
-	grid-template-columns: , 25%, 25%;
+	grid-template-columns: 25%, 25%, 25%;
 	font-weight: bold;
 	color: #2a2e32;
-	font-size: 2rem;
+	font-size: 1.5rem;
 }
 
 .portfolio-item {
```
Comment (maxwell-01) [inline style.css:163]: make this a flex container, set to column and align centre

File: style.css [hunk 16]
```diff
@@ -111,11 +142,12 @@ header div h1 {
 	height: 100vh;
```
Comment (maxwell-01) [inline style.css]: change to min-height

File: style.css [hunk 17]
```diff
@@ -136,14 +168,14 @@ header div h1 {
...
-	height: 100%;
+	height: auto;
 	width: 90%;
 	border-width: 5px;
 	border-radius: 10px;
 	padding: 10px;
 }
 
-.under-construction {
+.project-card-image {
```
Comment (maxwell-01) [inline style.css:177]: set to width %, eg `width: 60%;`

File: style.css [hunk 18]
```diff
@@ -12,8 +12,10 @@ html {
 	justify-content: center;
```
Comment (maxwell-01) [inline style.css:12]: the gap changes size due to your media query when the screen shrinks (see screenshots below). Consider using `gap: 20px` or another until you're happy with the result

image

image

**Review by maxwell-01 — CHANGES_REQUESTED** [review]

**Review by oritchie77 — APPROVED** [review]

**Pull request merged**